sdk/src/client/secret/placeholder.rs

```diff
@@ -3,68 +3,7 @@
 
 //! Implementation of [`PlaceholderSecretManager`].
 
-use std::ops::Range;
-
-use async_trait::async_trait;
-use crypto::{
-    keys::slip10::Chain,
-    signatures::secp256k1_ecdsa::{self, EvmAddress},
-};
-
-use super::{GenerateAddressOptions, SecretManage, SignTransactionEssence};
-use crate::{
-    client::{secret::PreparedTransactionData, Error},
-    types::block::{address::Ed25519Address, signature::Ed25519Signature, unlock::Unlocks},
-};
-
 /// Secret manager that is only useful to prevent accidental address generation in a wallet
 /// that has an offline counterpart for address generation and signing.
+#[derive(Debug)]
 pub struct PlaceholderSecretManager;
```

Comment:
Do we actually need this at all? It looks like just a unit variant of the enum would be enough.

Reply:
I tried removing it locally and it's fine 🤷🏻‍♂️

Reply:
🤨

Reply:
You don't have to remove it now, we can do in a separate PR because maybe it touches the bindings as well idk

Reply:
What do you mean I got it? Are you removing it now or later or not at all?

Reply:
Well it's still here ?